packages/anoncreds/src/services/AnonCredsHolderServiceOptions.ts

```diff
@@ -0,0 +1,102 @@
+import { CredentialInfo, RequestedCredentials } from '../models'
+
+import type {
+  AnonCredsCredential,
+  AnonCredsCredentialOffer,
+  AnonCredsCredentialRequest,
+  AnonCredsProofRequest,
+  NonRevokedInterval,
+  ReferentWalletQuery,
+} from '../models/exchange'
+
+import type {
+  AnonCredsCredentialDefinition,
+  AnonCredsRevocationList,
+  AnonCredsRevocationRegistryDefinition,
+  AnonCredsSchema,
+} from '../models/vdr'
+
+export interface AttributeInfo {
+  name?: string
+  names?: string[]
+}
+
+export interface CreateProofOptions {
+  proofRequest: AnonCredsProofRequest
+  requestedCredentials: RequestedCredentials
+  schemas: {
+    [schemaId: string]: AnonCredsSchema
+  }
+  credentialDefinitions: {
+    [credentialDefinitionId: string]: AnonCredsCredentialDefinition
+  }
+  revocationStates: {
+    [revocationRegistryDefinitionId: string]: {
+      definition: AnonCredsRevocationRegistryDefinition
+      revocationLists: {
+        [timestamp: string]: AnonCredsRevocationList
+      }
+    }
+  }
+}
+
+export type CreateProofMetadata = {
+  schemasMetaData: { [key: string]: { seqNo: number; ver: string } }
+  credentialDefinitionsMetaData: { [key: string]: { ver: string } }
+}
+
+export interface StoreCredentialOptions {
+  // TODO: what is in credential request metadata?
+  credentialRequestMetadata: Record<string, unknown>
+  credential: AnonCredsCredential
+  credentialDefinition: AnonCredsCredentialDefinition
+  credentialId?: string
+  revocationRegistryDefinition?: AnonCredsRevocationRegistryDefinition
+}
+export type StoreCredentialMetadata = {
+  credentialDefinitionMetadata: {
+    id: string
+    ver: string
+  }
+  revocationRegistryDefinition: {
+    id: string
+    issuanceType: 'ISSUANCE_BY_DEFAULT' | 'ISSUANCE_ON_DEMAND'
+    ver: string
```

Comment:
```suggestion
```

Reply:
Why did you remove the `issuanceType` here?

Reply:
Because it's not part of the spec anymore. The new revocation list model will contain 1s or 0s for all credential indexes, so you can infer the value from the state of revocation list. Indy will still use it and store it on ledger, but it's not needed for the anoncreds spec

Reply:
Can we hardcode it for Indy? The reason it's here is for Indy. But I get where the confusion comes from, all these metadata interfaces should be part of the indy-sdk package, not the anoncreds package.

Reply:
Even for indy it is in not needed on this layer. It will be needed to derie the revocation list model, but once we have that it doesn't matter which one we're using. If we need to go back to it we can just transform it to say ISSUANCE_BY_DEFUALT and align the deltas based on it (If all are revoked by default and we add all items to the detla that are unrevoked, or if all are unrevoked by default and we add the revoked credentials to the delta. The end result is the same -> we know which credentials are revoked)

Reply:
Hardcode to `ISSUANCE_BY_DEFUALT`